single-node-refactor/src/Solvers/SGH_solver/src/boundary.cpp

```diff
@@ -1,5 +1,5 @@
 /**********************************************************************************************
- � 2020. Triad National Security, LLC. All rights reserved.
+ � 2020. Triad National Security, LLC. All rights reserved.

```

Comment:
Im glad you caught this.  Ill look into updating this with something that doesnt get scrambled.